core/app/models/spree/refund.rb

```diff
@@ -29,26 +26,32 @@ def total_amount_reimbursed_for(reimbursement)
       end
     end
 
+    def perform_and_save!
+      perform!
+      save!
+      self
+    end
+
     def description
       payment.payment_method.name
     end
 
-    private
```

Comment:
Can we restore the `private` here please? Since everyone should call this via `perform_and_save!`.